packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -744,5 +744,6 @@ When registering an Inline Hook, you need to specify what type it is. The follow
 | `com.okta.oauth2.tokens.transform` | [Token Inline Hook](/docs/reference/token-hook/)               |
 | `com.okta.import.transform`        | [User Import Inline Hook](/docs/reference/import-hook/)        |
 | `com.okta.saml.tokens.transform`   | [SAML Assertion Inline Hook](/docs/reference/saml-hook/)       |
+| `com.okta.custom.source.delegated.authentication`   | [Custom Source Integration Delegated Authentication Inline Hook](/docs/reference/csi-delauth-hook/)       |
```

Comment:
```suggestion
```

Reply:
For beta releases, we don't add to this table (or the [supported Hooks table ](https://developer.okta.com/docs/concepts/inline-hooks/#currently-supported-types)in the Inline Hooks concepts. Only when the hook goes EA.